lib/commands/screenshots.js

```diff
@@ -42,51 +42,62 @@ async function getScreenshotWithIdevicelib (udid, isLandscape) {
 }
 
 async function isIdevicescreenshotAvailable () {
-  return !!(await fs.which('idevicescreenshot'));
+  try {
+    await fs.which('idevicescreenshot');
+  } catch (err) {
+    throw new Error(`No 'idevicescreenshot' program found. To use, install ` +
+                    `using 'brew install --HEAD libimobiledevice'`);
+  }
 }
 
 commands.getScreenshot = async function () {
+  const getScreenshotFromLIMD = async () => {
+    log.debug(`Taking screenshot with 'idevicescreenshot'`);
+    await isIdevicescreenshotAvailable();
+    const orientation = await this.proxyCommand('/orientation', 'GET');
+    return await getScreenshotWithIdevicelib(this.opts.udid, orientation === 'LANDSCAPE');
+  };
   const getScreenshotFromWDA = async () => {
+    log.debug(`Taking screenshot with WDA`);
     const data = await this.proxyCommand('/screenshot', 'GET');
     if (!_.isString(data)) {
       throw new Error(`Unable to take screenshot. WDA returned '${JSON.stringify(data)}'`);
     }
     return data;
   };
 
+  const useIdeviceScreenshot = this.opts.realDeviceScreenshotter &&
```

Comment:
you could also use `_.lowerCase(this.opts.realDeviceScreenshotter) === 'idevicescreenshot'`